dev/sg/sg_gen_access_token.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"github.com/urfave/cli/v2"
+
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/category"
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/std"
+	"github.com/sourcegraph/sourcegraph/internal/accesstoken"
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+)
+
+var (
+	genGatewayAccessTokenCommand = &cli.Command{
```

Comment:
```suggestion
	var codyGatewayCommand = &cli.Command{
```
Also, it would be better to make this a `cody-gateway` command with a subcommand `gen-token`
```go
	Name:     "cody-gateway",
	Usage:    "set of commands that are helpful for working with Cody Gateway locally",
	Category: category.Util,
	Subcommands: []*cli.Command{
		{
			Name:   "gen-token",
			Usage:  "generate a new token for use with Cody Gateway",
			Action: genGatewayAccessTokenExec,
		},
	},